### ISSUE TITLE
Returning JS is not interpreted

### ISSUE THREAD

**Issue by denispeplin** [jalkoby/phoenix_ujs#3]
I've added a link

```html
<%= link "⇧", to: post_upvote_path(@conn, :create, post), "ujs-method": "post", "ujs-remote": true %>
```

And `upvoted.js.eex` file
```js
console.log("upvoted!");
```

Server responds with `application/javascript; charset=utf-8`, but `console.log` doesn't work.

It is preferred way to get JS executed (like in Rails), but I've also tried to use `$(document).ajaxSuccess` and it doesn't work for me too.

**Comment by jalkoby** [comment]
Sorry, executing a js code from backend is not a good idea. It will not included into phoenix_js. I'm sorry once again

**Comment by jalkoby** [comment]
you can make a wrapper around phoenix_js which will eval response content

**Comment by denispeplin** [comment]
Thank you for quick reply. I've just assumed that `phoenix_ujs` will work like `jquery_ujs`, which does exactly this: executes code from backend. https://github.com/hauleth/vanilla-ujs does the same, however, Phoenix out of box doesn't have required html helpers to work with any of them.

Any information of how to make a wrapper around `phoenix_ujs` will be highly appreciated.

**Comment by jalkoby** [comment]
take a look here https://github.com/jalkoby/phoenix_ujs#things-which-are-not-included

**Comment by denispeplin** [comment]
Works fine, thanks!
Most my troubles were with `.ajaxSuccess()` not working, and I already started using `document.addEventListener('ajax:success', ...)`, but your refinement with `e.target` properties is very helpful.
What I found about `rails_ujs` is that it [encourages bad design](https://reinteractive.com/posts/195-rails-server-sent-javascript-considered-harmful), nothing about security issues.
Still I'm following your advice to not execute JS from server :)
I'm just need to catch `ajax:success` anyway, and this info was missing from Readme. Thank you again for adding it!
